src/server/utils/teamtailor.ts

```diff
@@ -92,7 +92,7 @@ export const initializeTeamtailorUsers = () =>
           e.status}", retrying after 5 sec...`,
       )
 
-      await new Promise((resolve) => setTimeout(resolve, 5000))
+      await new Promise((resolve) => setTimeout(resolve, 100))
```

Comment:
Why do we need change this? :)

Reply:
My mistake, shouldn't have been pushed 😅